docs/system-admin-guide/authentication/saml/README.md

```diff
@@ -171,6 +178,24 @@ You may provide attribute names or namespace URIs as follows: `email: ['http://s
 
 The OpenProject username is taken by default from the `email` attribute if no explicit login attribute is present.
 
+**b) Attribute mapping example for database**
+
+```ruby
+Setting.plugin_openproject_auth_saml = Hash(Setting.plugin_openproject_auth_saml).deep_merge({
+  "providers" => {
+    "saml" => {
+      "email" => "email",
+      "login" => "username",
+      "first_name" => "firstname",
+      "last_name" => "lastname"
```

Comment:
Even if it works, I'd add two attributes in an array here to show it is possible:

For example:
```
"login" => ['username', 'samAccountName', 'uid'],
```

Reply:
HI @oliverguenther 
i added your suggested changes, please verify them
Thank you